client/public/index.js

```diff
@@ -65,39 +65,41 @@ paypal
 
 paypal
   .PaymentFields({
-    fundingSource: paypal.FUNDING.IDEAL,
+    // style object is optional
```

Comment:
@jscheinhorn , isn't funding source needed ?

Reply:
ooh, thanks for catching that! deleted by mistake.